### ISSUE TITLE
Fix restangular url in setLogged

### ISSUE THREAD

**Issue by ptanov** [setelis/angular-se-authentication#4]
```
function setLogged(response) {
    service.currentLoggedMemberHolder.member = response;
    service.currentLoggedMemberHolder.logged = !!response;
    deferred.resolve(response);
```

vvvvvv
        // in config:
        // SeAjaxHelperService.fixRestangularUrl(response, $injector.get("MembersService").$membersOneRest(response.id));
^^^^
        return $q.when(response);
    }

**Comment by ptanov** [comment]
Blocked by #13

**Comment by ptanov** [comment]
Current way to do this:

``` javascript
SeAuthenticationService.addAuthenticationListener({
    onLogged: function(member) {
        SeAjaxHelperService.fixRestangularUrl(member.company, Restangular.one(REST.COMPANIES, member.company.id));
        });
    }
```
